test/Sema/accessibility_shared_private.swift

```diff
@@ -0,0 +1,76 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %utils/split_file.py -o %t %s
+// RUN: %target-swift-frontend -swift-version 4 -typecheck %t/declarations.swift %t/other_file_extensions.swift -verify
```

Comment:
Looks like that `@-1` syntax just changes the line, so I'm out of ideas how to make this test valid without 3 copies of this test =/